python/utils/OpaqueArguments.h

```diff
@@ -393,11 +393,11 @@ inline void packArgs(OpaqueArguments &argData, py::args args,
               .Case([&](IntegerType type) {
                 // Handle vec<bool> and vec<int>
                 if (type.getIntOrFloatBitWidth() == 1) {
-                  genericVecAllocator.template operator()<bool>(
+                  genericVecAllocator.template operator()<char>(
```

Comment:
Is this required if we merge #2338 ?

Reply:
The changes to this file can be dropped AFAICT.

It would be great to add the extended tests though.